qm-windowmanager/etc/systemd/session-active.container

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=session-activate container
+
+[Container]
+ContainerName=session-activate
+Environment=XDG_RUNTIME_DIR=/run/user/0
+Environment=DBUS_SESSION_BUS_ADDRESS=unix:path=/run/dbus/system_bus_socket
+Exec=/usr/bin/entrypoint.sh
+Image=session-activate:latest
+SecurityLabelType=qm_container_wayland_t
+Volume=/run/systemd:/run/systemd:ro
+Volume=/run/dbus/system_bus_socket:/run/dbus/system_bus_socket
+Volume=/run/user/0:/run/user/0
+
+[Install]
+WantedBy=multi-user.target
+
+[Service]
+Restart=always
```

Comment:
```suggestion
Restart=on-failure
```
Once the session is created, we do not need it to restart.

Actually, I am thinking that this could be perhaps be a `.service` instead of a Quadlet? We containerize workloads in the qm.container, but this is not a running workload, we just want to create the login session.

Reply:
sure, implemented as we talked.